Modules/Loadable/Markups/MRML/vtkMRMLMarkupsAngleNode.h

```diff
@@ -59,7 +67,47 @@ class  VTK_SLICER_MARKUPS_MODULE_MRML_EXPORT vtkMRMLMarkupsAngleNode : public vt
 
   /// Copy node content (excludes basic data, such as name and node references).
   /// \sa vtkMRMLNode::CopyContent
-  vtkMRMLCopyContentDefaultMacro(vtkMRMLMarkupsAngleNode);
+  vtkMRMLCopyContentMacro(vtkMRMLMarkupsAngleNode);
+
+  /// Get angle calculation mode, determining what angle to calculate from the given points.
+  /// \return \sa AngleMode (0:Unsigned, 1:OrientedSigned, 2:OrientedPositive)
```

Comment:
Enum values (0, 1, 2) are not part of the public API, they are internal implementation details that we should be able to change in the future without breaking backward compatibility. So, we should not include them in the API documentation.